pyvista/utilities/_getattr_.py

```diff
@@ -0,0 +1,41 @@
+"""Plotting utilities helper."""
+import importlib
+import inspect
+import warnings
+
+from pyvista.core.errors import PyVistaDeprecationWarning
+
+
+def __getattr__(name):
+    new_module_path = 'pyvista.core.utilities'
+    module = importlib.import_module(new_module_path)
+    try:
+        # First try the core utilities
+        value = inspect.getattr_static(module, name)
+        import_path = f'from {new_module_path} import {name}'
+    except AttributeError:
+        new_module_path = 'pyvista.plotting.utilities'
+        module = importlib.import_module(new_module_path)
+        try:
+            # Then try the plotting utilities
+            value = inspect.getattr_static(module, name)
+            import_path = f'from {new_module_path} import {name}'
+        except AttributeError:
+            new_module_path = 'pyvista'
+            module = importlib.import_module(new_module_path)
+            try:
+                # All else, try top level namespace
+                value = inspect.getattr_static(module, name)
+                import_path = f'from {new_module_path} import {name}'
+            except AttributeError:
+                raise AttributeError(
+                    f'Module `pyvista.utilities` has been deprecated and we could not automatically find `{name}` in `pyvista.core.utilities` or `pyvista.plotting.utilities` or the top level namespace.'
+                )
```

Comment:
Ah, I see my earlier suggestion here was stale... github showed me an earlier state.

Anyway, it's probably still relevant to suppress the exception context here:
```suggestion
                ) from None
```